src/integrationTest/java/uk/gov/hmcts/reform/pdf/service/integration/ResponseCodesTest.java

```diff
@@ -10,104 +10,108 @@
 
     private static final String API_URL = "/api/v1/pdf-generator/html";
+    private static final String PLACEHOLDER_VALUES = "placeholderValues";
+    private static final String NO_VALUES = "{ }";
+    private static final String TEMPLATE = "template";
 
     @Autowired
     private MockMvc webClient;
 
     @Test
     public void shouldReturn400WhenTemplateIsNotSent() throws Exception {
         webClient
-            .perform(fileUpload(API_URL)
-                .param("placeholderValues", "{ }"))
+            .perform(multipart(API_URL)
+                .param(PLACEHOLDER_VALUES, NO_VALUES))
             .andExpect(status().isBadRequest());
     }
 
     @Test
     public void shouldReturn400WhenPlaceholderValuesAreNotSent() throws Exception {
         webClient
-            .perform(fileUpload(API_URL)
-                .file("template", "<html></html>".getBytes(Charset.defaultCharset())))
+            .perform(multipart(API_URL)
+                .file(TEMPLATE, "<html></html>".getBytes(Charset.defaultCharset())))
             .andExpect(status().isBadRequest());
     }
 
     @Test
     public void shouldReturn400WhenEmptyTemplateIsSent() throws Exception {
         webClient
-            .perform(fileUpload(API_URL)
-                .file("template", new byte[] { })
-                .param("placeholderValues", "{ }"))
+            .perform(multipart(API_URL)
+                .file(TEMPLATE, new byte[]{})
+                .param(PLACEHOLDER_VALUES, NO_VALUES))
             .andExpect(status().isBadRequest());
     }
 
     @Test
     public void shouldReturn400WhenEmptyStringIsSentForPlaceholderValues() throws Exception {
         webClient
-            .perform(fileUpload(API_URL)
-                .file("template", "<html></html>".getBytes(Charset.defaultCharset()))
-                .param("placeholderValues", ""))
+            .perform(multipart(API_URL)
+                .file(TEMPLATE, "<html></html>".getBytes(Charset.defaultCharset()))
+                .param(PLACEHOLDER_VALUES, ""))
             .andExpect(status().isBadRequest());
     }
 
     @Test
     public void shouldReturn400WhenMalformedJsonIsSentForPlaceholderValues() throws Exception {
         webClient
-            .perform(fileUpload(API_URL)
-                .file("template", "<html></html>".getBytes(Charset.defaultCharset()))
-                .param("placeholderValues", "{:"))
+            .perform(multipart(API_URL)
+                .file(TEMPLATE, "<html></html>".getBytes(Charset.defaultCharset()))
+                .param(PLACEHOLDER_VALUES, "{:"))
             .andExpect(status().isBadRequest());
     }
 
     @Test
     public void shouldReturn400WhenMalformedHtmlTemplateIsSent() throws Exception {
         webClient
-            .perform(fileUpload(API_URL)
-                .file("template", "<html><not-html".getBytes(Charset.defaultCharset()))
-                .param("placeholderValues", "{ }"))
+            .perform(multipart(API_URL)
```

Comment:
Bonus for removing deprecated 🏅

Reply:
oops, that's not in that version, i just copied this out of my spring boot 2 PR, I'll remove from this PR and it'll come in the spring boot 2 one